extensions/amp-script/amp-script.md

```diff
@@ -301,25 +404,46 @@ The value of `max-age` should be chosen carefully:
 
 If you don't publish signed exchanges, `max-age` does nothing.
 
-**nodom (optional)**
-The `nodom` attribute optimizes `<amp-script>` for use as a data-layer rather than as a UI layer. It removes the ability for the `<amp-script>` to make DOM modifications, in favor of a signficantly smaller bundle size and therefore better performance. It also automatically hides the `<amp-script>`, so you may omit the height and width attributes.
-
-**common attributes**
+### common attributes
 
 This element includes [common attributes](https://amp.dev/documentation/guides-and-tutorials/learn/common_attributes) extended to AMP components.
 
-## Errors
+## Errors and warnings
 
-There are several types of runtime errors that may be encountered when using `amp-script`.
+A few runtime errors may be encountered when using `amp-script`.
 
-### "Maximum total script size exceeded (...)"
+**Inline script is (...) bytes, which exceeds the limit of 10,000."**
 
-`amp-script` limits the size of the JS source that may be used. See [Size of JavaScript code](#size-of-javascript-code) above.
+No inline script can exceed 10,000 bytes. See [Size of JavaScript code](#size-of-javascript-code) above. 
 
-### "Script hash not found."
+**Maximum total script size exceeded (...)**
+
+The total of all scripts used by a page cannot exceed 150,000 bytes. See [Size of JavaScript code](#size-of-javascript-code) above.
+
+**"Script hash not found."**
 
 For local scripts and cross-origin scripts, you need to add a [script hash](#script-hash) for security.
 
-### "amp-script... was terminated due to illegal mutation"
+**(...) must have "sha384-(...)" in meta[name="amp-script-src"]**
```

Comment:
Hmm, maybe we should remove the quotes from the outsides of bolded error messages? It's get complicated with attribute values

Reply:
Unless some have quotes and others don't?

Reply:
Good call. What a mess! I removed all the quotes for consistency.